data/gaming.html

```diff
@@ -262,9 +368,134 @@ <h2><a onclick="cmd('link?https://polychromatic.app/')"><span
+                                                class="material-icons">help</span></a>&zwnj;Mango HUD&zwnj;</h2>
+                                    <div class="card card-block">
+                                        <div class="card-image text-md-center">
+                                            <img src="img/screenshots/MangoHUD.gif" style="width: auto; max-height: 128px" />
+                                        </div>
+                                        <p class="card-text">&zwnj;A Vulkan and OpenGL overlay for monitoring FPS,
+                                            temperatures, CPU/GPU load and more.&zwnj;</p>
+                                    </div>
+                                    <div class="text-md-right">
+                                        <a id="mangohud-remove" class="btn btn-danger" onclick="cmd('remove?mangohud');">
+                                            <span class="material-icons">&#xE872;</span><span
+                                                class="button-text">&zwnj;Remove&zwnj;</span>
+                                        </a>
+                                        <a id="mangohud-install" class="btn btn-success hvr-shadow"
+                                            onclick="cmd('install?mangohud');"> &nbsp;
+                                            <span class="material-icons">&#xE2C0;</span>&zwnj;Install&zwnj;&nbsp;
+                                        </a>
+                                        <span id="mangohud-status" class="hidden">
+                                            <img src="img/welcome/processing.gif" width="24px" height="24px" />
+                                            &zwnj;Applying
+                                            Changes...&zwnj;
+                                        </span>
+                                        <p id="mangohud-instruction" class="hidden"><small>&zwnj;Please check the help
+                                                section for the Usage information&zwnj;</small></p>
+                                    </div>
+                                </div>
+                                <div class="col-md-6">
+                                    <br />
+                                    <h2><a onclick="cmd('link?https://gitlab.com/corectrl/corectrl')"><span
+                                                class="material-icons">help</span></a>&zwnj;CoreCtrl&zwnj;</h2>
+                                    <div class="card card-block">
+                                        <div class="card-image text-md-center">
+                                            <img src="img/logos/corectrl.png" style="width: auto; max-height: 128px" />
+                                        </div>
+                                        <p class="card-text">&zwnj;CoreCtrl allows you to control with ease your
+                                            computer hardware using application profiles.&zwnj;</p>
```

Comment:
Doesnt really inform the user - what are application profiles - and why should the gamer worry?  Maybe links here will help.

Reply:
Okay ... will clear this out. Its basically like configs

Reply:
Changed wording